integration/configs.go

```diff
@@ -22,10 +22,10 @@ const (
   schema: v9
   index:
     prefix: cortex_
-    period: 168h0m0s
```

Comment:
Why was this changed?

Reply:
Missed this when I changed `1w` to `168h` when I wanted older versions to run in integration tests: https://github.com/cortexproject/cortex/pull/2252/commits/d5f8f81a7a767149d22694bbe50f95ea58208081

PR to update changelog here: https://github.com/cortexproject/cortex/pull/2265